src/components/ViewComments.jsx

```diff
@@ -1,3 +1,54 @@
+import { useEffect } from "react"
+import { useState } from "react"
+import { useParams } from "react-router-dom"
+import { getComments } from "../api"
+
 const ViewComments =()=>{
-    
-}
\ No newline at end of file
+const[comments, setComments] = useState([])
+const[isLoading, setIsLoading] = useState(true)
+
+const {article_id} = useParams()
+
+const splitDate =(string)=>{
+        
+    let newDate = string.split("T")[0];
+ 
+    return newDate
+ }
+
+useEffect(()=>{
+    setIsLoading(true)
+    getComments(article_id).then(({comments})=>{
+        setComments(comments)
+        setIsLoading(false)
+    })
+},[article_id])
+
+if (isLoading) return <p>Loading...</p>;
+
+return (
+    <div className="commentSection">
+        <ul >
+            {comments.map((comment)=>{
+                return (
+                    
+                    <li className="comment"
+                    key={comment.article_id}
```

Comment:
this is the repeating key, so I'd suggest changing this value to something unique to each comment